workflows/files/ZWEWRF02.xml

```diff
@@ -79,6 +79,36 @@ MOUNT FILESYSTEM('${zfsdsn}') +
 TYPE(ZFS) MODE(RDWR) +
 PARM('AGGRGROW') +
 MOUNTPOINT('${zfsmount}')
+/*</inlineTemplate>
+            <submitAs maxRc="8">JCL</submitAs>
+            <maxLrecl>80</maxLrecl>
+        </template>
+    </step>
+  <step name="chown" optional="false">
+        <title>Change ownership</title>
+        <description>This step changes the ownership and mode of ZOWE USS directory so the zwe command can be called.</description>
+        <instructions substitution="false">Make sure you have correct SYSAFF in the following JCL so the filesystem is
```

Comment:
Make sure you have the correct SYSAFF in the following JCL so the filesystem is
         mounted correctly.